packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -146,9 +146,7 @@ export class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardStat
     );
   }
 
-  private _getTargetElement(): HTMLElement | undefined {
-    const { target } = this.props;
-
+  private _getTargetElement(target: HTMLElement | string | null | undefined): HTMLElement | undefined {
```

Comment:
nit: You could use `target?` instead of adding `undefined` as one of the possible types.